stable/distribution/templates/distribution-statefulset.yaml

```diff
@@ -124,8 +124,8 @@ spec:
         - name: distribution-data
           mountPath: {{ .Values.distribution.persistence.mountPath | quote }}
       {{- end }}
-    {{- if .Values.distribution.customInitContainers }}
-{{ tpl .Values.distribution.customInitContainers . | indent 6}}
+    {{- if .Values.common.customInitContainers }}
+{{ tpl .Values.common.customInitContainers . | indent 6}}
```

Comment:
Use with instead of if please, let's try to use with as a convention for these sort of statements

Reply:
Done